src/providers/data_provider/dp_targets.c

```diff
@@ -304,16 +239,6 @@ static errno_t dp_target_special(struct be_ctx *be_ctx,
         }
     }
 
-    if (target->target == DPT_SUDO) {
-        if (dp_target_sudo_enabled(be_ctx)) {
-            return EAGAIN;
```

Comment:
That's something you'll have to argue with @pbrezina as I'm just following his suggestion here.